lib/Sema/CSGen.cpp

```diff
@@ -24,6 +24,7 @@
 #include "swift/AST/PrettyStackTrace.h"
 #include "swift/AST/SubstitutionMap.h"
 #include "swift/Sema/IDETypeChecking.h"
+#include "swift/Subsystems.h"
```

Comment:
Why did this change?

Reply:
I moved createTypeChecker() from IDETypeChecking.h (which is still used in this file) to Subsystems.h.